package/network/utils/iptables/Makefile

```diff
@@ -454,17 +454,17 @@ endef
 
 define Package/ip6tables-nft
 $(call Package/iptables/Default)
-  DEPENDS:=ip6tables @IPTABLES_NFTABLES +libxtables-nft
+  DEPENDS:=@IPV6 iptables-nft +kmod-nf-ipt6
   TITLE:=IP firewall administration tool nft
 endef
 
 define Package/ip6tables-nft/description
 Extra ip6tables nftables nft binaries.
-  iptables-nft
-  iptables-nft-restore
-  iptables-nft-save
-  iptables-translate
-  iptables-restore-translate
+  ip6tables-nft
+  ip6tables-nft-restore
+  ip6tables-nft-save
+  ip6tables-translate
+  ip6tables-restore-translate
```

Comment:
Am I wrong or is this an untreated fixup?

Reply:
moved to another commit